utils/s2n_mem.c

```diff
@@ -293,6 +293,16 @@ int s2n_free(struct s2n_blob *b)
     return S2N_SUCCESS;
 }
 
+int s2n_free(struct s2n_blob *b)
+{
+    return s2n_free_impl(b, true);
+}
+
+int s2n_free_non_zeroed(struct s2n_blob *b)
+{
+    return s2n_free_impl(b, false);
+}
```

Comment:
I'm not a fan of this name. For an embarrassingly long time thought this method assumed that the memory hadn't been zeroed and therefore would zero it, not the opposite. (This feedback applies to the stuffer operations too-- that was where my confusion started)

Other possible options: s2n_free_unsafe, s2n_free_encrypted, s2n_free_public? Or super explicit: s2n_free_without_wipe

Reply:
I think `s2n_free_without_wipe` is probably the clearest on what it does.